### ISSUE TITLE
a11y: OpenstaxCNX logo needs alt text denoting what the image is

### ISSUE THREAD

**Issue by brittweinstein** [Connexions/webview#1758]
@brittweinstein commented on [Thu Jul 26 2018](https://github.com/Connexions/content-managers/issues/164)

From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

Logo image of Openstax CNX is not labeled as an image:

Suggestion: label logo as an image with a link to the main page.

![image.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/e57563d0-e4c0-4fa5-b64d-be3d47e1ea39)

**Comment by Stackblocks** [comment]
The logo now has alt text reading "OpenStax CNX" but from an accessibility standpoint it should probably be more along the lines of "Openstax CNX Logo and link to main page" or something like that for people who are operating their computer with limited or no vision. Is that correct @brittweinstein ?

![image.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/9f3def4b-0470-4ffd-9d89-f0cf5b9d0a8b)

**Comment by brittweinstein** [comment]
I think @openstaxalina would be the best person to make this call!

**Comment by openstaxalina** [comment]
@Stackblocks good point.  I reviewed the "Logos" section here: https://webaim.org/techniques/alttext/

And determined that we can leave this as-is.  "OpenStax CNX" is correct in this case.
